src/index.js

```diff
@@ -22,6 +22,7 @@ export default (
     onExamineFile,
     onCheckFile,
     onWriteFile,
+    resolveConfig = true,
```

Comment:
You said to default this to false but from your wording it sounded like we wanted resolving the config to be the default behavior and if this is defaulted to false, I would have to negate it in `isSupportedExtension.js` to determine whether or not to resolve the config, which seemed confusing

Reply:
Yep you got it right.